### ISSUE TITLE
Domain attic-backup.org expired, pending delete

### ISSUE THREAD

**Issue by jniggemann** [jborg/attic#401]
Name: attic-backup.org
Registry Domain ID: D171289523-LROR
Domain Status: pendingDelete, serverHold, redemptionPeriod
Registry Expiration: 2021-03-04 11:51:14 UTC

**Comment by adept** [comment]
attic is dead, long live borg?

**Comment by jniggemann** [comment]
Indeed :-)

**Comment by fhavrlent** [comment]
I found attic in some blog post, got linked to the website... I was wondering what is going on 😃

**Comment by jgrisham** [comment]
_For the information of future travelers stumbling here:_
* The owner of this project is on a multi-year sabbatical _(due to discovering that time with family and children is awesome yet also a time-suck)_.

<hr />

### What I've learned in the past hour*
_(* likely to be quite incomplete)_

#### Last archived version of attic-backup.org
* [attic-backup.org](http://web.archive.org/web/20210302202145/https://attic-backup.org/) _(via Internet Archive, 2021-03-02)_
  * [Earlier snapshot](http://web.archive.org/web/20190424084232/https://attic-backup.org/index.html) _(same content)_ that won't erroneously link to new, unrelated site archives

#### Forks:
* This project was _(manually?)_ forked several years ago to [borgbackup/borg](https://github.com/borgbackup/borg) _([documentation site](https://borgbackup.readthedocs.io/))_.
  * It can import existing [jborg/attic](https://github.com/jborg/attic) backup repositories.
    * _**(which will then no longer backwards compatible)**_
  * `borgbackup` appears to be under active development with a community, as of early 2022.
* @AaronWebster mentioned a few months back _(in jborg/attic#403)_ that he was interested in maintaining a direct and compatible fork of `attic`.
  * <s>[AaronWebster/attic](https://github.com/AaronWebster/attic) has recent commits but does not currently appear to have GitHub issue reporting enabled</s>.
  * [AaronWebster/attic](https://github.com/AaronWebster/attic) is the [most recently-updated direct fork](https://github.com/jborg/attic/network) as of February 2022

#### Similar _(but independently-inspired)_ projects:
* [bup/bup](https://github.com/bup/bup)
  * Under active development, with a mailing list and IRC channel.
  * Based on git packfile format
  * Exists in several package managers.
  * _(see also related tools, e.g. [anarcat/bup-cron](https://github.com/anarcat/bup-cron))_
 * ? 

<hr />

### Tl;dr:
1. If you wish to stay compatible with the `attic` format,
    * [AaronWebster/attic](https://github.com/AaronWebster/attic) appears to be the [only active direct fork](https://github.com/jborg/attic/network) as of February 2022
2. If you don't have legacy needs, 
    * `bup` and `borgbackup` both have 'watching' and 'fork' numbers of the same order of magnitude.
      * _(I have **no** idea which is more popular/in greater use, more stable, more efficient, etc.)_
    * There may be better alternatives elsewhere, or perhaps there are not. :)

<hr />

P.S. Sincere and hearty **thanks to Jonas** _(aka @jborg)_ for both the initial creation of `attic` and for sharing it with the world.
* It appears to have had quite a broad impact _(to end users and to other developers)_ and for _(inadvertantly?)_ helping to advance both the popularity and the state of the art in open-source deduplicated backup tools!

**Comment by AaronWebster** [comment]
I enabled issue reporting and am happy to review/manage PRs going forward as this is something I actively use.
